tasks/eth/fp-recovery/001-blacklist-game/README.md

```diff
@@ -0,0 +1,64 @@
+# Deputy Guardian - Blacklist Dispute Game
+
+Status: CONTINGENCY TASK, SIGN AS NEEDED
+
+## Objective
+
+This task executes a `blacklistDisputeGame` operation in the `OptimismPortalProxy` on behalf of the `Guardian` role. Blacklisting the dispute game prevents any withdrawals proven to be contained within the game's proposed output root from being finalized.
+
+The batch will be executed on chain ID `1`, and contains `1` transactions.
+
+## Tx #1: Blacklist Dispute Game
+
+Executes the `blacklistDisputeGame` call to the `OptimismPortalProxy`.
+
+**Function Signature:** `blacklistDisputeGame(address,address)`
+
+**To:** `0x4220C5deD9dC2C8a8366e684B098094790C72d3c`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x629cdd49000000000000000000000000<OptimismPortalProxyAddress------------>000000000000000000000000<DisputeGameToBlacklist---------------->`
```

Comment:
Maybe we can this make clearer with a tab? 

|Function Selector Signature| OptimismPortalProxyAddress (32 bytes with "0" on the left before address) | DisputeGameToBlacklistAddress  (32 bytes with "0" on the left before address)|
|--------------------------|--------------------------------------------|--------------------------------------------|
|0x629cdd49|000000000000000000000000`OptimismPortalProxyAddress`|000000000000000000000000`DisputeGameToBlacklistAddress`|

Reply:
Added